### PR TITLE
Update to PowerToys Awake documentation

### PR DESCRIPTION
**Pull request by dend**

### CODE REVIEW

File: hub/powertoys/awake.md [hunk 1]
```diff
@@ -1,66 +1,74 @@
...
 
 ## Getting started
 
-You can use PowerToys Awake directly from PowerToys Settings or as a standalone executable. When it's running from PowerToys, it can be managed from PowerToys Settings or the system tray.
+You can use PowerToys Awake directly from PowerToys Settings or as a standalone executable (`PowerToys.Awake.exe` in the PowerToys installation folder).
 
 > [!NOTE]
-> PowerToys Awake does not modify any of the Windows power plan settings, and does not depend on a custom power plan configuration. Instead, it spawns background threads that tell Windows that they require a specific state of the machine.
+> PowerToys Awake does not modify any of the Windows power plan settings and does not depend on a custom power plan configuration. Instead, it spawns background threads that tell Windows that they require a specific state of the machine. Once PowerToys Awake exists, the threads are terminated and the computer will resume its standard power plan behavior.
```
Comment (alvinashcraft) [inline hub/powertoys/awake.md]: Should "exists" be "exits"?

**Review by alvinashcraft — COMMENTED** [review]
@dend this looks good to me, except for the minor issue I commented on. I think it should exits instead of exists.

File: hub/powertoys/awake.md (hunk 1 above)
Comment (dend) [reply hub/powertoys/awake.md]:
```suggestion
> PowerToys Awake does not modify any of the Windows power plan settings and does not depend on a custom power plan configuration. Instead, it spawns background threads that tell Windows that they require a specific state of the machine. Once PowerToys Awake exits, the threads are terminated and the computer will resume its standard power plan behavior.
```

**Comment by dend** [comment]
@alvinashcraft good catch - I've just updated it. Should be good now!

**Review by alvinashcraft — APPROVED** [review]

**Pull request merged**